docs/examples/advanced/fullstack-with-react-apps

```diff
@@ -0,0 +1,33 @@
+---
+id: fullstack-with-react-apps
+title: "Creating Full Stack Apps with React and Zio Http"
+sidebar_label: "Serve React App"
+---
+
+```scala mdoc
+import zio._
+import zio.http._
+import zio.stream.ZStream
+
+import java.io.File
+import java.nio.file.Paths
+
+object HelloWorld extends ZIOAppDefault {
+    // Create the build relative directory path
+    private val buildDirectory = "./test"
+    
+    // Create HTTP route
+    val app = Http.collectHttp[Request] {
+        case Method.GET -> Root / "api" / "hello" => Handler.text("Hello World!").toHttp
+
+        // Uses netty's capability to write file content to the Channel
+        // Content-type response headers are automatically identified and added
+        // Adds content-length header and does not use Chunked transfer encoding
+        case Method.GET -> Root => Http.fromFile(new File(s"$buildDirectory/index.html"))
+        case Method.GET -> "" /: file => Http.fromFile(new File(s"$buildDirectory/$file"))
```

Comment:
Don't we need to protect against `..`? Someone asking for `../../aFile` could try to download system files, no?

Reply:
File secrets in unsafe in current logic, right?

Reply:
What do you call "File secrets"?

Reply:
Mounted secrets in kubernetes/Docker apps

Reply:
I'm not a 100% sure, someone would need to give a try but I think that if we don't protect against the `..` file access, any file in the file system is accessible to someone having access to the API

Reply:
@guizmaii Can you share in which URL would be able to go through ".."? That path from the URL won't translate as is in the app I believe. PFA a recording of the same.

https://github.com/zio/zio-http/assets/46300090/fd9c2ea0-5e49-4b3c-846f-1744b6a13481

Reply:
@rishi-raj-jain Can you try with curl, please? Someone malicious will use a script to try to steal data

Reply:
PFA

https://github.com/zio/zio-http/assets/46300090/97b037f6-471a-4d8b-b3b5-b81a1a2eab4f

Reply:
What does "PFA" means?

Reply:
Hey guizmaii, oh actually I'm used to writing that. it's please find attached. I was just referring to the video that I attached

Reply:
@guizmaii 

Everything looking good here?

Reply:
@rishi-raj-jain The ticket #2292 asks for an example project combining React and zio-http.
I'm not working for Ziverge but I'm not sure you can claim the prize with just some documentation

Reply:
@guizmaii 

That's what I've shown in the videos and the demo. But yeah a place to upload the project would be fine.

> just

Oops